packages/react/src/components/checkbox/checkbox.stories.tsx

```diff
@@ -19,6 +22,7 @@ export default {
     },
   },
   args: {
+    children: '',
```

Comment:
🤔  curious, would not setting anything here be any different?

Reply:
Very little - Storybook adds `-`:

<img width="1030" alt="Screenshot 2021-02-11 at 10 40 30" src="https://user-images.githubusercontent.com/20243687/107626384-9b646500-6c55-11eb-9888-0736dc8fd1ee.png">

I'm not entirely sure why, but I have noticed this some time ago also with some other story.

Reply:
Ah, it's probably because it's not sure of the type.
If we set it under `argTypes` it probably changes to `string` or whatever.
Anyway, looks good as is.